test/any/web_catalog/api-extractor-test.es6.js

```diff
@@ -0,0 +1,331 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+(function() {
+  let objectGraph = global.objectGraph;
+  let extractor = com.web.catalog.apiExtractor.create({});
+  const types = {
+    'boolean': 2,
+    'exception': 7,
+    'null': 6,
+    'number': 3,
+    'string': 4,
+    'symbol': 5,
+    'undefined': 1,
+  };
+
+  describe('Method getObjectProperties', function() {
```

Comment:
Maybe we could nest describes here. One for `API Extractor`, and then an inner describe for each method. WDYT?